README.md

```diff
@@ -1,5 +1,13 @@
 # Unicop
 
+## Usage
+
+```sh,ignore
+unicop [FILES]...
+```
+
+Where `[FILES]...` is a list of files or directory to check, default: `.`.
```

Comment:
I think we should default to printing the CLI help output when no argument is given, not default to the current directory. This can be changed later. But I think it is better to be more explicit. And also that you get help rather than start an expensive job when you run without arguments. It's not like it's hard to manually invoke it with `unicop .` if that's what you want. And it's more clear.

Examples of other tools that does recursive scanning of directories:
* `osv-scanner` - prints help when no arguments given
* `cloc` - prints help when no arguments given
* `find` - defaults to `find .` when no arguments are given
* `ls` - defaults to current directory when no arguments given